extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonRecorder.java

```diff
@@ -16,7 +16,8 @@ public JacksonConfigSupport get() {
                         jacksonBuildTimeConfig.failOnEmptyBeans,
                         jacksonBuildTimeConfig.writeDatesAsTimestamps,
                         jacksonBuildTimeConfig.acceptCaseInsensitiveEnums,
-                        jacksonBuildTimeConfig.timezone.orElse(null));
+                        jacksonBuildTimeConfig.timezone.orElse(null),
+                        jacksonBuildTimeConfig.serializationInclusion.orElse(null));
```

Comment:
I wonder why we don't take advantage of the very descriptive `Optional` and pass it on (instead of having to check for `null` in the producer), but maybe that's to prevent lamdas (`serializationInclusion.ifPresent(...)`)?

Anyway, it was there before (`timezone`) so feel free to ignore this comment.

Reply:
Yes, AFAIR the intention was to avoid lambdas here